### ISSUE TITLE
Periodically can't update spot_spotnodepool in place

### ISSUE THREAD

**Issue by MarcPerezPro** [rackerlabs/terraform-provider-spot#8]
# Issue
Sometimes when updating in place the `bid_price` of a `spot_spotnodepool` I get an error like:

> │ Error: Provider produced inconsistent result after apply
> │ 
> │ When applying changes to
> │ module.bid["us-east-iad-1--ch.vs1.2xlarge-iad"].spot_spotnodepool.bid[0],
> │ provider "provider[\"registry.terraform.io/rackerlabs/spot\"]" produced an
> │ unexpected new value: .resource_version: was cty.StringVal("7061731"), but
> │ now cty.StringVal("7063735").
> │ 
> │ This is a bug in the provider, which should be reported in the provider's
> │ own issue tracker.

The issue usually goes away after 30 minutes, but it happens a few times a day.

# Workaround
Here is my workaround to force recreation instead of updating in place:
```hcl
# Workaround to force the recreation of the spotnodepool when the bid price changes
resource "null_resource" "force_recreate_when_bid_price_changes" {
  triggers = {
    bid_price = local.bid_price
  }
}

resource "spot_spotnodepool" "bid" {
  cloudspace_name      = var.cloudspace_name
  server_class         = var.server_class
  bid_price            = local.bid_price
  desired_server_count = var.desired_server_count
  lifecycle {
    replace_triggered_by = [
      null_resource.force_recreate_when_bid_price_changes
    ]
  }
}
```

**Comment by TheNilesh** [comment]
A fix is currently in progress to remove the `resource_version` attribute, as it is not particularly useful to users.

Try `terraform refresh` as immediate fix.

**Comment by TheNilesh** [comment]
@MarcPerezPro The fix has been pushed in v0.0.8. Could you please confirm if that resolves your issue?

**Comment by MarcPerezPro** [comment]
@TheNilesh That fixed my issue, it works without the workaround
